src/sentry/sentry_metrics/indexer/strings.py

```diff
@@ -87,6 +87,7 @@
     "d:transactions/alert@none": PREFIX + 133,
     "s:transactions/alert@none": PREFIX + 134,
     "g:transactions/alert@none": PREFIX + 135,
+    "d:transactions/duration_coarse@millisecond": PREFIX + 136,
```

Comment:
Ahh sorry I should've said this in our meeting, but I wonder if we should name this more explicitly? In starfish we have `exclusive_time_light` but I'm thinking if we add another "light" metric the name instantly becomes bad.

Reply:
hmmm `duration_low_cardinality`?

Reply:
lol i'm a big fan of very long descriptive names so I'm tempted to say like
`duration_with_only_transaction_tag` but maybe you can get more feedback from the relay team in https://github.com/getsentry/relay/pull/2513

Reply:
I'll check with @jan-auer when he's back

Reply:
ok yeah we've landed on `duration_light` to match `exclusive_time_light` based on Jan's feedback